screens/home_screens/HotScreen.js

```diff
@@ -81,18 +82,68 @@ const hot_events = [
   
 
 export default class HotScreen extends React.Component {
+  state = {
+    posts:[ ]
+  }
+  componentDidMount(){
+      axios.get('https://quiet-spire-38612.herokuapp.com/api/events/5c607e3c8149f900160c89f6')
+        .then(res => {
+          this.setState({
+            posts: res.data.slice(1,10)
```

Comment:
see above comment - the reason why the slice doesn't work is because you are trying to call slice on an object, this should work after adjusting which API endpoint you are hitting